src/SignalRServiceExtension/Bindings/EndpointConnectionInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    internal class EndpointConnectionInfo
+    {
+        public ServiceEndpoint ServiceEndpoint { get; set; }
+        public string Url { get; set; }
+        public string AccessToken { get; set; }
```

Comment:
is it auto generate josn object to for js cx? 
then if cx returns this to client, the client will see connection string? since `ServiceEndpoint` contains it.
or we can control it somewhere?

Reply:
looks like if this instance is returned to client in response, function app admin need to generate a new object only contains `url` and `accessToken` to avoid secret leak. then function app admin have to have the knowledge about negotiation response, and ask us what to set about `ConnectionId` and `AvailableTransports`. maybe a property only for  `url` and `accessToken`  is fine, admin can simply respond it. @vicancy @Y-Sindo

Reply:
Not all of the properties in `ServiceEndpoint` will be serialized, see https://github.com/Azure/azure-functions-signalrservice-extension/blob/3f33c997848ace6d5d6b2c2ca11195168a8f92d7/src/SignalRServiceExtension/Bindings/ServiceEndpointJsonConverter.cs#L10 . 
But I think we can consider whether wrapping url and accessToken in a separate object would be convenient for users.